### PR TITLE
fix: polkadot v1.1.0 client

### PR DESCRIPTION
**Pull request by wischli**
# Description

Context: https://github.com/centrifuge/centrifuge-chain/pull/1756/files#r1562566212

* Deprecates `parachain-id` CLI argument which was a custom addition for a few CLI arguments inherited from Substrate
  * Sets para id `2000` for ephemeral chain specs `development`, `centrifuge-local`, `altair-local`
  * Removes defaulting to `10001` para id if not set in CLI
* Removes deprecated chain specs `charcoal`, `algol`, `centrifuge-dev` (basically `centrifuge-local with 2 more funded accounts) and `altair-dev` (basically `altair-local` with 2 more funded accounts)

Fixes #(issue)

* Local parachain deploymentg

# Checklist:

- [x] I have added Rust doc comments to structs, enums, traits and functions
- [x] I have made corresponding changes to the documentation
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by lemunozm — APPROVED** [review]
Thanks so much for dealing with this @wischli! And for updating the docs related too!

**Comment by lemunozm** [comment]
Just a couple of questions. 
- Do you want to merge this after merging `polkadot-v1.1.0` or before?
- Is catalyst still needed?

**Comment by wischli** [comment]
> * Is catalyst still needed?

For now, it is still used so I would keep it. But its expected to be deprecated sometime this year!

File: node/src/command.rs
Comment (lemunozm) [inline node/src/command.rs:84]: I think we've broken our development benchmarking with this. We didn't have a CI job for development (I do not remember why), and we didn't notice it.

We should call it `development-local` or modify `check_benchmarks.sh` script.

cc @wischli

**Pull request merged**
